ios/REAModule.m

```diff
@@ -156,6 +156,7 @@ - (void)addOperationBlock:(AnimatedOperation)operation
 
 - (void)uiManagerWillPerformMounting:(RCTUIManager *)uiManager
 {
+  [_nodesManager tryToFlushUpdateBuffer];
```

Comment:
I still think it'd be better if we could run these updates on UI thread w/o using RCTExecuteOnMainQueue which would postpone these updates until the next frame